eng/common/Update-Change-Log.ps1

```diff
@@ -12,7 +12,8 @@ param (
   [Parameter(Mandatory = $true)]
   [String]$ChangeLogPath,
   [String]$Unreleased = $True,
-  [String]$ReplaceVersion = $False
+  [String]$ReplaceVersion = $False,
+  $ReleaseDate
```

Comment:
Should declare as `[DateTime]`. This will validate various formats.

Reply:
It's a string in reality in a changelog format. We parse it later.